pkg/prometheus/promcfg_test.go

```diff
@@ -3271,7 +3271,138 @@ func TestRemoteWriteConfig(t *testing.T) {
 					},
 				},
 			},
-			golden: "RemoteWriteConfig_v2.45.0_1.golden",
```

Comment:
Any special reason to change the golden file of the previous test? I believe we still want to test version 2.45.0 with the 2.45.0 golden file 🤔

Reply:
My mistake!